README.md

```diff
@@ -11,47 +11,72 @@ and feature-complete.
 
 ## Getting started
 
-### Installation
+Use uvx to get started quickly:
 
 ```shell
-uv tool install ty@latest
+uvx ty check
 ```
 
-or add ty to your project:
+### Adding ty to your project
+
+Use [uv](https://github.com/astral-sh/uv) (or your project manager of choice) to ty as a development
+dependency in your project:
 
 ```shell
 uv add --dev ty
+```
 
-# With pip.
-pip install ty
+Adding ty as a dependency ensures that all developers on the project are using the same version of
+ty.
+
+Next, use `uv run` to invoke ty:
+
+```shell
+uv run ty
 ```
 
-### First steps
+### Installation
 
-After installing ty, you can check that ty is available by running the `ty` command:
+Install ty with uv:
 
 ```shell
-ty
+uv tool install ty@latest
+```
+
+Or, install with pip:
+
+```shell
+pip install ty
+```
+
+After installing ty, check that ty is available by running the `ty` command:
+
+```console
+$ ty
 An extremely fast Python type checker.
 
 Usage: ty <COMMAND>
 
 ...
 ```
 
-You should see a help menu listing the available commands.
-
-For detailed information about command-line options, see the [commands] reference.
+You should see a help menu listing the available commands. For detailed information about command-line options, see the [commands] reference.
 
 ## Checking a project
 
-The easiest way to type check your project is by running ty through [uv](https://docs.astral.sh/uv/). To do that, add ty as a project dependency:
```

Comment:
This was sort of intentional to use `uv run` over `uvx` here because, as far as I understand, uvx doesn't set the virtual environment automatically. Meaning, your out-of-the-box experience is worse.

Reply:
I'm exploring that, but the experience should be about equivalent with a `.venv` because ty detects that? `uv run` just implements discovery of `.venv` in parents, so I don't think the experience should be much different.

Since it's alpha, I think we might want to encourage the ephemeral experience over "make this a dependency of the project for other developers".

I'm just trying things out here right now though.

Reply:
I'm fine either way. I only wanted to point out why I went with `uv run` over `uvx`

Reply:
I think I'm going to prefer `uvx` here because I don't want to encourage people to add it as a dependency (yet) and after a few iterations I think it's cleaner to avoid multiple installation methods in the README itself.